netlify.toml

```diff
@@ -26,3 +26,9 @@
   for = "/*.bash"
   [headers.values]
     content-type = "text/x-shellscript"
+
+[[redirects]]
+  from = "/latest/*"
+  to = "/:splat"
```

Comment:
For anyone else unfamiliar with netlify https://docs.netlify.com/routing/redirects/redirect-options/#splats